brave-spancollector-scribe/src/main/java/com/github/kristofa/brave/scribe/ScribeSpanCollectorNotificationsHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.github.kristofa.brave.scribe;
+
+import com.twitter.zipkin.gen.Span;
+
+/**
+ * Monitor scribe span collector's performance by implementing reactions to these notifications,
+ * e.g. by updating some metrics in the metrics tool used in your application.
+ */
+public interface ScribeSpanCollectorNotificationsHandler {
+
+    /**
+     * Called when span queue size changes.
+     *
+     * @param currentQueueSize the current queue size.
+     */
+    void onQueueSizeChanged(int currentQueueSize);
+
+    /**
+     * Called when a span could not be added to the span queue because of it being full
+     * and so the span was rejected.
+     *
+     * @param rejectedSpan the rejected span.
+     */
+    void onSpanRejected(Span rejectedSpan);
```

Comment:
ps probably don't need the Span parameter to increment a counter. ex `incrementDroppedSpans()` or if we want to call out only dropped spans due to queue rejection (vs marshaling, timeouts or other reasons), `incrementRejectedSpans()`